### PR TITLE
Refine dsl components samples

### PR DESCRIPTION
**Pull request by 0mza987**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

File: sdk/jobs/pipelines/2a_train_mnist_with_tensorflow/train_mnist_with_tensorflow.ipynb [hunk 1]
```diff
@@ -128,7 +128,7 @@
     "    description=\"Train using TF component\",\n",
     ")\n",
     "def sample_pipeline():\n",
-    "    tf_job = tf_func(epochs=1)\n",
+    "    tf_job = train_tf(epochs=1)\n",
     "    tf_job.compute = \"cpu-cluster\"\n",
     "    tf_job.instance_count=2\n",
```
Comment (wangchao1230) [inline sdk/jobs/pipelines/2a_train_mnist_with_tensorflow/train_mnist_with_tensorflow.ipynb:133]: instance_count is not correct

**Comment by wangchao1230** [comment]
Add the CI pipeline configs

File: sdk/jobs/pipelines/1b_pipeline_with_python_function_components/pipeline_with_python_function_components.ipynb [hunk 2]
```diff
@@ -122,8 +122,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# train_component_func = ml_client.components.create_or_update(train_component_func)\n",
-    "print(train_component_func)"
+    "# train_model = ml_client.components.create_or_update(train_model)\n",
```
Comment (wangchao1230) [inline sdk/jobs/pipelines/1b_pipeline_with_python_function_components/pipeline_with_python_function_components.ipynb:125]: no need to have the create section in this sample

Comment (0mza987) [reply sdk/jobs/pipelines/1b_pipeline_with_python_function_components/pipeline_with_python_function_components.ipynb]: removed

File: sdk/jobs/pipelines/2a_train_mnist_with_tensorflow/train_mnist_with_tensorflow.ipynb (hunk 1 above)
Comment (0mza987) [reply sdk/jobs/pipelines/2a_train_mnist_with_tensorflow/train_mnist_with_tensorflow.ipynb]: updated

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**